setup/docker/compose.yml

```diff
@@ -40,7 +41,8 @@ services:
 
   pulsar:
     image: apachepulsar/pulsar:3.0.0
-    profiles: [ "test", "dev", "pulsar" ]
+    container_name: pulsar
+    profiles: [ "test", "dev", "pulsar" , "dev-metric"]
```

Comment:
i think we can include the prometheus into the test and dev profile by default. I dont think metric specific profile makes sense.

Reply:
possibly yes, might need to tune the resources here otherwise laptop becomes issue. Will see if this can be done (might keep same for now).